### DISCUSSION TITLE
Feature Freeze

### DISCUSSION THREAD

**Discussion by cserver45** [01000011-00110111/School-Chat#147] [Ideas]
I propose we put in effect a feature freeze and finish all currently open issues, before abandoning this project.

If you look at the activity of this repo, you will see the amount of commits over the past 2 months has decreased significantly.
I like to leave my projects that I start finished, so I will hope we do the same here.

**Comment by cserver45** [comment]
Seems like we will be going with this for the most part

**Reply by 01000011-00110111** [reply]
We are only going to add issues if they are needed. So any new ideas will be left to the side for now

**Reply by 01000011-00110111** [reply]
How about we put new ideas and features here

**Reply by cserver45** [reply]
We could create a new thread for that, I kind of want to keep this just for news on the feature freeze

**Reply by 01000011-00110111** [reply]
That would work

**Comment by cserver45** [comment]
I was wrong last month, looks like we have reversed the decision on this (sigh)

**Reply by 01000011-00110111** [reply]
hard not to, so well try our best to slow them down

**Comment by 01000011-00110111** [comment]
got 3 issues left

**Comment by 01000011-00110111** [comment]
we did it finally. 2 issues left and 1 of them is almost done

**Comment by 01000011-00110111** [comment]
and for the foreseeable future this will be in effect as we are behind on adding features back aka mostly me needing to make private chats work again.
